static/scss/style.scss

```
@@ -716,6 +717,25 @@ body.showing-search {
             &.active {
                 background: #1a1919;
             }
+
+            .search-title {
+                padding-top: 12px;
+                font-size: 20px;
+            }
+
+            .text-matches {
+                padding-top: 8px;
+                padding-bottom: 10px;
+                font-size: 14px;
+
+                .search-result-match {
+                    color: black;
+                    background-color: yellow;
+                    border-radius: 3px;
+                    padding: 1px 2px;
+                    border: 1px solid #ecb430;
```

Won't this need a light variant?
```suggestion
                    color: black;
                    background-color: yellow;
                    border-radius: 3px;
                    padding: 1px 2px;
                    border: 1px solid #ecb430;
```

---

Nevermind, wouldn't make sense here 😅